# HackBio_Genomics_Workshop-Week_3

## Standing on the Shoulders of Giants
The week's task was to replicate a tutorial on Identification of Somatic and Germline Variants from Tumor and Normal Sampple Pairs through variant calling and extensive variant annotation.  
The tutorial was divided into 6 main steps:
1. Data pre-processing
2. Read mapping
3. Mapped reads post-processing
4. Variant calling and classification
5. Variant annotation
6. Variant Reporting

Data pre-processing to variant calling stage was carried out in Linux pipeline on the HackBio server. However due to space constraint, the tutorial was completed on Galaxy Server. The codes for the Linux workflow can be found in stage2.sh and a full report of the task is available in Stage2_Report.pdf above.

## Reference Tutorials
1. https://github.com/Fredrick-Kakembo/Somatic-and-Germline-variant-Identification-from-Tumor-and-normal-Sample-Pairs
2. https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/somatic-variants/tutorial.html#removing-duplicate-reads

Thank you!
